apis/config/v1alpha1/types_cert_manager.go

```diff
@@ -2,11 +2,13 @@ package v1alpha1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+// CertManager specifies cluster-wide settings for cert-manager.
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+//+kubebuilder:resource:scope=Cluster
```

Comment:
These should not be necessary, I do not see it in openshift/api. @sttts ?

Reply:
That's an interesting thing... I had to add it in order to generate proper, cluster scoped CRD.

Reply:
With the Makefile change above, this should not be necessary